src/main/java/com/kodilla/ecommercee/controller/GroupController.java

```diff
@@ -0,0 +1,32 @@
+package com.kodilla.ecommercee.controller;
+
+import com.kodilla.ecommercee.domain.GenericEntity;
```

Comment:
Wydaje mi się, że nie przechodzi Ci testu CirleCi dlatego, że moje zmiany nie zostały zatwierdzone jeszcze przez @hubert-ros i przez to w Main GenericEntity nie jest jeszcze przeniesiony do pakietu domain. 

Na razie możesz zostawić jak jest, bo w końcu będzie to przeniesione i testy Ci przejdą. Ewentualnie dla potestowania możesz zmienić tą linijkę na import com.kodilla.ecommercee.GenericEntity; i zobaczymy czy mam rację 😉

Reply:
Może być tak jak Tomek piszę. Dlatego zmiany na swoich branchach powinny być zawsze robione w odniesieniu do `main`, a nie do jeszcze nie zmergowanych pull requestów